doc/api/http2.md

```diff
@@ -740,6 +739,21 @@ argument identifying the frame type, and an integer argument identifying the
 error code. The `Http2Stream` instance will be destroyed immediately after the
 `'frameError'` event is emitted.
 
+#### Event: `slowRead`
+<!-- YAML
+added: REPLACEME
+-->
+
+The `'slowRead'` event is emitted when a connected peer takes too long to
+consume the queued data. This is typically evidence of a flow control issue
+or a slow read DOS attack. By default, the event will be emitted if the
```

Comment:
I do not think we should mention a DoS attack here, as this mechanism alone is not sufficient to protect us from this.